src/routes/(app)/admin[[hash=admin_hash]]/order/+page.server.ts

```diff
@@ -15,6 +24,13 @@ export async function load({ url }) {
 			})
 			.sort({ createdAt: -1 })
 			.toArray();
+	} else if (paymentMethod) {
+		orders = await collections.orders
+			.find({
+				'payments.method': paymentMethod
+			})
+			.sort({ createdAt: -1 })
+			.toArray();
 	} else {
```

Comment:
The parameters should be able to be combined, eg skip + payment method together. You can default `skip` to `0` if it's helpful